de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5512210_sys_gh4941_AD_Process_IsTranslateExcelHeaders.sql

```diff
@@ -0,0 +1,230 @@
+-- 2019-02-07T17:32:19.179
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Element (AD_Client_ID,AD_Element_ID,AD_Org_ID,ColumnName,Created,CreatedBy,EntityType,IsActive,Name,PrintName,Updated,UpdatedBy) VALUES (0,576095,0,'IsTranslateExcelHeaders',TO_TIMESTAMP('2019-02-07 17:32:18','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','Translate Excel Headers','Translate Excel Headers',TO_TIMESTAMP('2019-02-07 17:32:18','YYYY-MM-DD HH24:MI:SS'),100)
```

Comment:
suggested description:
EN: If checked, then metasfresh will replace the header name with a translation, if one exists in Message (AD_Message) or Element (AD_Element).

DE: Wenn angehakt, dann wird metasfresh die jeweiligen Spaltenüberschriften durch Übersetzungen ersetzen, sofern welche in Meldung (AD_Message) oder Element (AD_Element) vorhanden sind.

Reply:
Done